validation/validationhelper/numeric.go

```diff
@@ -0,0 +1,17 @@
+// Package validationhelper provides validation helper functions for govalid.
+package validationhelper
+
+// IsNumeric checks if a string contains only digit characters (0–9).
+func IsNumeric(s string) bool {
+	if s == "" {
+		return false
+	}
+
+	for _, ch := range s {
+		if ch < '0' || ch > '9' {
+			return false
+		}
+	}
+
+	return true
+}
```

Comment:
So -42 is not numeric ?
Same for 3.14 ?

alphanumeric, that is commonly accepted and understood being [A-Za-z0-9]

But numeric sounds like numbers. And number can be float, negative...

So maybe "numeric" is not the best name

It might be OK but I'm raising the point just in case